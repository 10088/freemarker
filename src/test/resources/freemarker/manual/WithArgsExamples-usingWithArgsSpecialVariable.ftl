<#macro m1 a b c>
  m1 does things with ${a}, ${b}, ${c}
</#macro>

<#macro m2 a b c>
  m2 does things with ${a}, ${b}, ${c}
  Delegate to m1:
  <@m1?with_args(.args) />
</#macro>

<@m2 a=1 b=2 c=3 />
