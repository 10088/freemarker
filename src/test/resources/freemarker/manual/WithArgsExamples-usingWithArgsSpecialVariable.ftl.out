
  m2 does things with 1, 2, 3
  Delegate to m1:
  m1 does things with 1, 2, 3
